core/src/test/java/kafka/test/server/BootstrapControllersIntegrationTest.java

```diff
@@ -235,4 +246,55 @@ private void testIncrementalAlterConfigs(ClusterInstance clusterInstance, boolea
             });
         }
     }
+
+    @ClusterTest
+    public void testAlterReassignmentsWithBootstrapControllers(ClusterInstance clusterInstance) throws ExecutionException, InterruptedException {
+        String topicName = "foo";
+        try (Admin admin = Admin.create(adminConfig(clusterInstance, false))) {
+            Map<Integer, List<Integer>> assignments = new HashMap<>();
+            assignments.put(0, Arrays.asList(0, 1, 2));
```

Comment:
this manual assignments requires "3" brokers, but the test starts with only one broker, right?